php-practice/section5.php

```diff
@@ -320,3 +338,25 @@ public function send() {
  *   'cost' => 3.99
  * ];
  */
+
+class ShoppingCart {
+  private $item;
+  private $shoppingCart;
+
+  $item = [
+    'name' => 'Pringles',
+    'cost' => 3.99
+  ];
+
+  public function addItem() {
+    $shoppingCart =  [];
```

Comment:
This resets the list of items to an empty array every time we add an item.

But by doing that, we're basically erasing all of the items previously added. So that's not something we want!